modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/SwiftCodegen.java

```diff
@@ -115,10 +117,10 @@ public SwiftCodegen() {
         typeMapping.put("string", "String");
         typeMapping.put("char", "Character");
         typeMapping.put("short", "Int");
```

Comment:
I found `typeMapping.put("short", "Short");` in `public DefaultCodegen()`, and it is overridden in `public SwiftCodegen()`
So maybe keep the line but change to `typeMapping.put("short", "Int16");` ?